packages/dx-react-chart-material-ui/package.json

```diff
@@ -60,32 +60,32 @@
     "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-filenames": "^1.2.0",
+    "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-jest": "^21.17.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.8.2",
+    "eslint-plugin-react": "^7.9.1",
     "jest": "^23.1.0",
-    "jss": "^9.8.1",
+    "jss": "^9.8.6",
     "jss-preset-default": "^4.5.0",
     "jss-theme-reactor": "^0.11.1",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0",
-    "react-test-renderer": "^16.4.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-test-renderer": "^16.4.1",
     "rollup": "0.50.0",
     "rollup-plugin-babel": "^3.0.4",
     "rollup-plugin-license": "^0.6.0",
     "rollup-plugin-node-resolve": "^3.3.0"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "prop-types": "^15.6.1"
+    "classnames": "^2.2.6",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "@devexpress/dx-chart-core": "1.3.0",
     "@devexpress/dx-react-chart": "1.3.0",
     "@devexpress/dx-react-core": "1.3.0",
-    "@material-ui/core": "^1.1.0",
+    "@material-ui/core": "^1.2.3",
```

Comment:
Is it needed to update minimal required version of mui?

Reply:
Probably not